plugins/available/percol.plguin.bash

```diff
@@ -0,0 +1,13 @@
+# Install
+## (sudo) pip install percol
+## bash-it enable percol
+
+# Usage
+## C-r to search&select from history
+
+_replace_by_history() {
```

Comment:
I think it would make sense to add a check to see whether `percol` is installed. If someone enables this plugin without having `percol` installed, the key binding will still happen, but the command will fail. Can you add an `if` around the function and the `bind` command to only be run when the `percol` command is available?